skimage/measure/_find_contours_cy.pyx

```diff
@@ -93,6 +103,13 @@ def iterate_and_store(double[:, :] array,
             coords[0] += 1
             coords[1] = 0
 
+        if _nodata_enabled:
+            if npy_isnan(_nd):
+                if (npy_isnan(ul) or npy_isnan(ur) or npy_isnan(ll) or npy_isnan(lr)):
```

Comment:
Why the enclosing brackets here?

Reply:
I originally wrote this whole block as one big boolean expression before thinking better of it, so the parens are a hangover from that. I'll get rid of them.

Reply:
Fixed by 6bd1ae2.